src/apps/chifra/cmd/serve/server.go

```diff
@@ -0,0 +1,16 @@
+package serve
+
+import "net/http"
+
+// TODO: this should be soon part of pkg/utils
```

Comment:
Yes. Or the package `colors`, or (maybe better) `colors` should be combined with `utils`. Your call.